src/Microsoft.ServiceFabric.Actors/Client/ActorProxy.cs

```diff
@@ -27,8 +27,10 @@ public abstract class ActorProxy : ProxyBase, IActorProxy
         private RemotingClientVersion remotingClient;
 
 #if !DotNetCoreClr
+#pragma warning disable 618

```

Comment:
I think I've seen using Obsolete attribute for private fields instead of suppressing warning 618. Is there an advantage to doing it either way? Let's be consistent.